.github/workflows/test-distribution.yml

```diff
@@ -104,6 +91,11 @@ jobs:
         run: |
           apt-get -y install clang-14
 
+      - name: Install supported compiler
+        if: ${{ matrix.os == 'ubuntu:24.04' }}
+        run: |
+          apt-get -y install clang-18
```

Comment:
The `linux-arm64` config uses `matrix.compiler` value, might be useful here as well.